src/core/server/saved_objects/migrations/core/opensearch_index.test.ts

```diff
@@ -292,21 +303,24 @@ describe('ElasticIndex', () => {
     test('throws error if re-index task fails', async () => {
       client.indices.getAlias.mockResolvedValue(
         opensearchClientMock.createSuccessTransportRequestPromise({
-          '.my-fanci-index': '.muchacha',
+          '.my-fanci-index': { aliases: { '.muchacha': {} } },
         })
       );
       client.reindex.mockResolvedValue(
-        opensearchClientMock.createSuccessTransportRequestPromise({ task: 'abc' })
+        opensearchClientMock.createSuccessTransportRequestPromise({
+          task: 'abc',
+        } as opensearchtypes.ReindexResponse)
       );
       client.tasks.get.mockResolvedValue(
+        // @ts-expect-error @opensearch-project/opensearch GetTaskResponse requires a `task` property even on errors
```

Comment:
Is there a follow-up needed here?

Reply:
[issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1360) is created